_posts/2023-01-22-auto-restarting-connectors.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: post
+title:  "Auto-restarting connectors in Apache Kafka Connect and Mirror Maker 2"
+date: 2023-01-22
+author: jakub_scholz
+---
+
+Apache Kafka Connect provides a framework for integrating Apache Kafka with external systems using connectors.
+Connectors run inside the Kafka Connect deployment, connect to the external system, and help to push data from Kafka to the external system or the other way around.
+When running Kafka Connect, you have to monitor not only the state of the Kafka Connect deployment, but also the state of the connectors and their tasks.
+If a connector or one of its tasks fail, you check the reason it failed so you know how to fix it.
```

Comment:
Should we recap how you can check if anything has failed and if so find the reason?

Reply:
TBH, I do not want to get into the details of that. But if you would be interested to write a blog post about it it would definitely find many readers 😉.

Reply:
Fair enough. Yes this could be an interesting blog.